test/app/submissions_test.rb

```diff
@@ -52,6 +52,12 @@ def test_guests_can_view_submissions
     assert_response_status(200)
   end
 
+  def test_guests_get_redirected_on_nitpick
```

Comment:
Whoops! Good call, I missed that somehow. I’ll remove the spec.

I’m trying to decide the best way to spec out the whole comment -> login -> redirect process.

> On Apr 27, 2016, at 11:54 AM, Amalrik Maia <notifications@github.com> wrote:
> 
> In test/app/submissions_test.rb <https://github.com/exercism/exercism.io/pull/2849#discussion_r61284545>:
> 
> > @@ -52,6 +52,12 @@ def test_guests_can_view_submissions
> >      assert_response_status(200)
> >    end
> >  
> > +  def test_guests_get_redirected_on_nitpick
> I think this is duplicating test_guest_nitpicks.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/exercism/exercism.io/pull/2849/files/0d11c21bb0c5c7a0b1b5755f4c95686700cb0bbe#r61284545>